lib/options-manager.js

```diff
@@ -29,7 +29,7 @@ const DEFAULT_EXTENSION = [
 const DEFAULT_CONFIG = {
 	useEslintrc: false,
 	cache: true,
-	cacheLocation: path.join(os.homedir() || os.tmpdir(), '.xo-cache/'),
+	cacheLocation: findCacheDir({name: 'xo'}),
```

Comment:
Can `xo` be run against a stand-alone script (IE not part of any module)?  If `find-cache-dir` fails to find any package.json then it will return `null`.  If needed I can re-add the original cacheLocation value as a fallback, but no point if `xo` cannot be run in that manner.

Reply:
> Can xo be run against a stand-alone script (IE not part of any module)?

Yes

Reply:
Fixed